torch/_inductor/compile_fx.py

```diff
@@ -317,6 +317,7 @@ def fake_tensor_prop(
     The created fake mode will be returned.
     """
     fake_mode = detect_fake_mode(example_inputs)
+    fake_mode.allow_non_fake_inputs = True
```

Comment:
is it ok to always run with allow_non_fake_inputs?

Reply:
Is it because we have mixed fake/non-fake inputs so that we have to set this flat to be True? If so, where do the non-fake inputs come from? Thanks.

Reply:
Removed this line.